docs/tutorials/embedding/MASE.ipynb

```diff
@@ -0,0 +1,377 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Embed the two graphs using MASE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Just as ASE fits a single input graph to the Random Dot Product Graph (RDPG) model, MASE fits a sample of graphs to the common subspace independent-edge (COSIE) model, which is an extension of RDPG for multiple graphs. The COSIE model assumes that all of the graphs have important shared properties, so all graphs are derived from a common subspace with orthonormal basis $V$.  But, COSIE also allows the connection probabilities between each vertex to vary from one graph to the next, so each graph $A^{(i)}$ has a corresponding score matrix $R^{(i)}$.  As a result, COSIE facilitates comparison due to the common subspace, while retaining sufficient flexibility to represent heterogeneous collections of graphs, due to the different score matrices."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we know that 2 clusters should be embedded into 2 dimensions, we specify the number of components as 2. Then, the ``fit_transform()`` method returns the estimated invariant subspace $\\hat{V}$, which we can see has 50 entries corresponding to the 50 vertices embedded in 2 dimensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.embed import MultipleASE as MASE\n",
+    "\n",
+    "embedder = MASE(n_components=2)\n",
+    "V_hat = embedder.fit_transform([G1, G2])\n",
+    "print(V_hat.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize Common Latent Subspace"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since the two graphs have clear block structures, with higher within-block probabilities than between-block, we see two distinct \"clusters\" when we visualize the latent positions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(10,10))\n",
+    "ax.scatter(V_hat[:25, 0], V_hat[:25, 1], c=\"blue\", label=\"Block 1\")\n",
+    "ax.scatter(V_hat[25:, 0], V_hat[25:, 1], c=\"red\", label=\"Block 2\")\n",
+    "ax.legend()\n",
+    "_ = ax.set_title(\"Common Latent Subspace from MASE Embedding\", fontsize=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Recover Probability Matrices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since ``MultipleASE()`` stores estimates of the individual score matrices $\\hat{R}^{(i)}$ as well as an estimate of the common invariant subspace $\\hat{V}$, we can recover an estimate of the probability matrix for each graph \n",
```

Comment:
fyi, you used P in two different meanings in this tutorial. as matrix in K \cross K, and as a matrix in n \cross n.

Reply:
Yes, I was following the example of the omnibus tutorial, but Giorgio just pointed out today that it isn't good usage, and somewhat confusing.  Will fix.